api/api-config-repos-v4/src/main/java/com/thoughtworks/go/apiv4/configrepos/ConfigReposControllerV4.java

```diff
@@ -57,16 +57,18 @@
 
 @ToggleRegisterLatest(controllerPath = Routes.ConfigRepos.BASE, apiVersion = ApiVersion.v4, as = "branch_support")
 @Component
-public class ConfigReposControllerV4 extends ApiController implements SparkSpringController, CrudController<ConfigRepoConfig> {
+public class ConfigReposControllerV4 extends AbstractMaterialTestController implements SparkSpringController, CrudController<ConfigRepoConfig> {
```

Comment:
Are you planning to document the test connection endpoint? I remember there was a conversation earlier if we should document the test connection API or keep it internal. I think it was decided to keep it internal as the assumption was it will be used only on the SPA. If we are planning to keep it internal we should not add the endpoint in this controller.

Reply:
@maheshp - Yes, this needs to be an internal API. I will move it to internal config repos api